public/app/features/alerting/unified/utils/templates.ts

```diff
@@ -12,3 +12,9 @@ export function ensureDefine(templateName: string, templateContent: string): str
   }
   return content;
 }
+export function updateAndSanitizeDefine(newDefine: string, templateContent: string): string {
```

Comment:
Could you add some docs here? A few words of explanation of what this function does